java-frontend/src/test/files/se/SeEngineTest.java

```diff
@@ -31,6 +31,7 @@ void test_null_assignement() {
 
   void local_variable() {
     Object a;
+    a = null;
```

Comment:
wondering if this should be fixed or removed altogether given the testcase added.